ovc5/software/camera_device_driver/include/ovc5_driver/camera.hpp

```diff
@@ -1,11 +1,11 @@
 #ifndef CAMERA_INC
 #define CAMERA_INC
-#include <string>
-#include <memory>
-
 #include <ovc5_driver/i2c_driver.h>
 #include <ovc5_driver/vdma_driver.h>
 
+#include <memory>
+#include <string>
```

Comment:
Yeah I fixed those in my other PR but I didn't modify any text in this change. Only ran the formatting script.

Reply:
We should still try to keep all the formatting to this PR and all the functionality to the other though, rather than having half formatted code and fixing all the remaining issues in the feature PR.